inc/saihostif.h

```diff
@@ -409,6 +409,13 @@ typedef enum _sai_hostif_trap_type_t
      */
     SAI_HOSTIF_TRAP_TYPE_MPLS_ROUTER_ALERT_LABEL = 0x00007004,
 
+    /**
+     * @brief MPLS packet with label trapped to CPU
```

Comment:
What's the use case for this trap. Sorry if this has been covered earlier, still catching up with videos.